packages/mobile/src/utils/time.ts

```diff
@@ -275,16 +277,19 @@ export const getDatetimeDisplayString = (timestamp: number, i18next: i18nType) =
 
 export const getRemoteTime = async () => {
   const getNetworkTime = promisify(ntpClient.getNetworkTime)
+  ntpClient.ntpReplyTimeout = 2000
```

Comment:
In places with bad connection having the timeout be 2 seconds will most likely not be enough and we'll see a lot more timeouts.

IIRC the problem with having a bigger drift is that the light client will fail, right? I'm guessing very few people are using the light client and I don't think it works well anyways, so it might make sense to leave this like it was and change the drift tolerance to be 21 seconds instead.


As a side note, in the future if both these requests time out we need to retry at some point in the future, otherwise this is not doing anything really.

Reply:
That was my initial thinking also but looking back at why @tarikbellamine added the ntp check in the first place it looks like fetching the pepper might want more accurate timestamps: https://github.com/celo-org/wallet/commit/bd5c02e51b66daa37dfb4536fec23057c6daa225#diff-a37126ff1aba3601ad5b60575833ea09d2484d1d816acc3938a7e8bd775cf9c8

I don't know how accurate it wants it to be though. Any ideas?

Reply:
I will just make this 21 seconds as you suggested for now. As we learn more about what each service requires we can revisit this.

Reply:
We could use something like [ts-retry-promise](https://www.npmjs.com/package/ts-retry-promise) to handle the retries with a backoff.